components/prombench/manifests/benchmark/1c_cluster-role-binding.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: loadgen-scaler
+  name: loadgen-scaler-{{ .PR_NUMBER }}
```

Comment:
Since this is a clusterrolebinding, it is not namespaced.
If the {{ .PR_NUMBER }} is not used, then on deploying multiple prombench instances, previous loadgen-scaler rules will be modified to new ones.

Reply:
> previous loadgen-scaler rules will be modified to new ones.

Why is this a problem?

Reply:
I had created loadgen-scaler as a cluster-role, because I used to deploy the cluster using gcloud. But now since everything uses the same service-account.json file, I can change it to namespace-role.
Then  {{ .PR_NUMBER }} won't be needed

Reply:
nice , I like it when we can simplify :stuck_out_tongue_winking_eye: